src/app/components/ThemeProvider/withThemeProvider.tsx

```diff
@@ -124,6 +124,11 @@ import {
 import gridWidths from './gridWidths';
 
 import { BrandPalette, Typography, BrandSVG } from '../../models/types/theming';
+import { PageTypes } from '../../models/types/global';
+
+const isDarkUiPage = (pageType: PageTypes, derivedPageType?: string | null) =>
+  pageType === 'mediaArticle' ||
+  (pageType === 'media' && derivedPageType?.toLowerCase() === 'on demand tv');
```

Comment:
Possibly overkill for now, but one for the future, could we have an array of supported dark UI page types?

 ```suggestion
 const darkUIPages = ['on demand tv'];
  pageType === 'mediaArticle' ||
  (pageType === 'media' && darkUIPages.includes(derivedPageType?.toLowerCase());
```

Reply:
Yea I think if we start supporting more dark UI pages then we should do something like this 👍